umap/views.py

```diff
@@ -275,7 +275,7 @@ def get_object(self):
 
     def get_maps(self):
         qs = self.get_search_queryset() or Map.objects.all()
-        qs = qs.filter(Q(owner=self.object) | Q(editors=self.object))
+        qs = qs.filter(owner=self.object).union(qs.filter(editors=self.object))
```

Comment:
```suggestion
        qs = qs.union(qs.filter(owner=self.object), qs.filter(editors=self.object))
```

This version is more readable to me, not a strong opinion though.